tests/Oryx.Integration.Tests/NodeEndToEndTests.cs

```diff
@@ -255,35 +255,18 @@ public async Task CanBuildAndRun_NodeApp(string nodeVersion)
         }
 
         [Theory]
-        [InlineData("oryxdevms/pack-builder")]
-        [InlineData("heroku/buildpacks")]
+        [InlineData(Constants.OryxBuildpackBuilderImageName)]
+        [InlineData(Constants.HerokuBuildpackBuilderImageName)]
         public async Task CanBuildAndRun_NodeApp_WithBuildpack(string builder)
```

Comment:
move these two tests (CanBuildAndRun_NodeApp_WithBuildpack and TwigExample_WithBuildpack ) under a new class something like 
```
public class NodeBuildPackTests: NodeEndToEndTestsBase
{
}
```

Node integration tests takes very long, this way we can optimize a little

Reply:
yeah, I meant TwigExample_WithBuildpack  to move to a new class in php ernd to end tests